src/main/java/uk/co/tfn/HelperMethods.java

```diff
@@ -433,33 +438,97 @@ public void completeUserDetailsPage() {
     }
 
     public void randomlyDetermineUserType() {
-        int randomSelector = randomNumberBetweenOneAnd(2);
-        switch (randomSelector) {
-            case 1:
-                // 1. Click Any and continue
-                this.clickElementById("passenger-type-0");
-                this.continueButtonClick();
-                this.waitForPageToLoad();
-                break;
-            case 2:
-                // 2. Click a non-Any, complete the next page, and continue
-                int randomUserType = randomNumberBetweenOneAnd(6);
+        waitForPageToLoad();
+        int randomSelector = randomNumberBetweenOneAnd(3);
 
-                WebElement element = this
-                        .waitForElement(String.format("passenger-type-%s", String.valueOf(randomUserType)));
+        if(randomSelector == 1){
+            // Click Any and continue
+            WebElement element = this.waitForElement("passenger-type-anyone");
+            if (this.browser.equals("ie")) {
+                JavascriptExecutor executor = (JavascriptExecutor) this.driver;
+                executor.executeScript("arguments[0].click();", element);
+            } else {
+                element.click();
+            }
```

Comment:
Looks like we use this chunk of code a fair few times - would it be worth splitting it out into it's own function?

Reply:
Done!